nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/resources/META-INF/services/org.apache.nifi.controller.ControllerService

```diff
@@ -19,3 +19,4 @@ org.apache.nifi.services.azure.storage.ADLSCredentialsControllerService
 org.apache.nifi.services.azure.cosmos.document.AzureCosmosDBClientService
 org.apache.nifi.services.azure.storage.AzureStorageEmulatorCredentialsControllerService
 org.apache.nifi.services.azure.storage.AzureStorageCredentialsControllerService_v12
+org.apache.nifi.services.azure.StandardAzureCredentialsControllerService
```

Comment:
A bit unfortunate that the existing standard here for Azure extensions seems to be to group everything in the `nifi-azure-processors` module.  I think ideally we'd want a separate `nifi-azure-service-api` and `nifi-azure-service-api-nar`.  However, since the precedent is to store Azure services in this module, I think you're right to have it here for now.  Perhaps this lends itself to a restructuring for NiFi 2.0!